redirector/requirements.txt

```diff
@@ -1,8 +1,65 @@
+#
+# This file is autogenerated by pip-compile
```

Comment:
pip-tools FTW!

Reply:
yes, they are pretty neat. I got the tip from you in basket, thanks!